examples/src/main/java/ai/djl/examples/inference/SemanticSegmentation.java

```diff
@@ -66,28 +66,18 @@ public static void predict() throws IOException, ModelException, TranslateExcept
                         .addTransform(new Normalize(mean, std))
                         .build();
 
-        Criteria<Image, Image> criteria =
+        Criteria<Image, Segmentation> criteria =
                 Criteria.builder()
                         .optApplication(Application.CV.SEMANTIC_SEGMENTATION)
-                        .setTypes(Image.class, Image.class)
+                        .setTypes(Image.class, Segmentation.class)
                         .optModelUrls(url)
                         .optTranslator(translator)
                         .optEngine("PyTorch")
                         .optProgress(new ProgressBar())
                         .build();
-        try (ZooModel<Image, Image> model = criteria.loadModel();
-                Predictor<Image, Image> predictor = model.newPredictor()) {
-            Image semanticImage = predictor.predict(img);
-            saveSemanticImage(semanticImage);
+        try (ZooModel<Image, Segmentation> model = criteria.loadModel();
+                Predictor<Image, Segmentation> predictor = model.newPredictor()) {
+            return predictor.predict(img);
         }
     }
-
-    private static void saveSemanticImage(Image img) throws IOException {
```

Comment:
We still want to save segmented image, and we can save a few different mode:
1. only colors
2. transparent color overlay on existing image
3. replace background with other image